Automattic/pocket-casts-android
# Playback 2023 - Top five podcasts story

| 📘 Part of: #1463 | 🎨 Designs: lH66LwxxgG8btQ8NrM0ldx-fi-1599_20385 |
|:---:|:---:|

## Description
Adds the 2023 visual for the Top Podcasts Story.


## Testing Instructions
1. Make sure you're logged in to an account that has a few episodes listened
2. Go to Profile
3. Tap the EOY image
4. Skip to the fourth story
5. ✅ Check that the story looks good
6. Share this story
7. ✅ The story image should look good

## Screenshots or Screencast 

Pixel 7 
|-----|
| <img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/b313d50d-7f6a-4475-9f46-9ec6d7fb2e56"/> |

Nexus 9 Tablet
|-----|
| <img width=400 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/7cc2deba-0849-481a-881e-cf483b884f1c"/> | 


## Checklist
 
#### I have tested any UI changes...
